gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/GenerateRestClientTask.java

```diff
@@ -221,9 +224,9 @@ public void generate()
           }
           javaExecSpec.setClasspath(_pathedCodegenClasspath);
           javaExecSpec.setMain("com.linkedin.restli.tools.clientgen.FluentApiGenerator");
-          javaExecSpec.args("-resolverPath", resolverPathArg);
-          javaExecSpec.args("-rootPath", getProject().getRootDir().getPath());
-          javaExecSpec.args("-targetDir", _destinationDir.getAbsolutePath());
+          javaExecSpec.args("--resolverPath", resolverPathArg);
```

Comment:
Does this mean it didnt work when we do just one dash?

Reply:
yeah, this was a recent code, and we never enabled the task before.